java/examples/webhook-server/README.md

```diff
@@ -0,0 +1,23 @@
+# Java webhook server example
+
+A http server than can receive and verify signed TrueLayer webhooks.
+
+## Run
+
+Run the server.
+
+```sh
+./gradlew run
```

Comment:
webhook-server examples works fine for me :+1: just the same diff for running the server.

```suggestion
../../gradlew run
```